components/supervisor/pkg/supervisor/tasks.go

```diff
@@ -506,7 +506,7 @@ func (r *loggingHeadlessTaskProgressReporter) write(data string, task *task, ter
 
 func (r *loggingHeadlessTaskProgressReporter) done(success bool) {
 	workspaceLog := log.WithField("component", "workspace")
-	workspaceLog.WithField("taskLogMsg", taskLogMessage{Type: "workspaceTaskOutput", Data: "🚛 uploading prebuilt workspace"}).Info()
+	workspaceLog.WithField("taskLogMsg", taskLogMessage{Type: "workspaceTaskOutput", Data: "🚛 uploading workspace image"}).Info()
```

Comment:
actually neither of those two messages is correct. In case of prebuilds the new message is wrong, incase of image builds the old one is.

Reply:
@csweichel why the old message is correct for normal builds?